superset/db_engine_specs/__init__.py

```diff
@@ -49,3 +50,6 @@
             and attribute.engine != ""
         ):
             engines[attribute.engine] = attribute
+            if attribute.engine in _translates:
+                _dialect = _translates[attribute.engine]
+                engines[_dialect] = attribute
```

Comment:
..Here we can then do a loop over the aliases, mapping them to the original engine class.

Reply:
Nice suggestion! I will implement this soon.